src/api/views.py

```diff
@@ -496,4 +513,161 @@ def check_license(request):
+        code = re.search(r'code=([a-zA-Z0-9]+)',currentUrl.encode('utf-8')).group(1)
+        token = generate_github_access_token(githubClientId, githubClientSecret, code)
+        clientId = settings.OAUTHTOOLKIT_APP_CLIENT_ID
+        clientSecret = settings.OAUTHTOOLKIT_APP_CLIENT_SECRET
+        backend = settings.BACKEND
+        djangoToken = convert_to_auth_token(clientId, clientSecret, backend, token)
+        try:
+            user = get_user_from_token(djangoToken)
+        except TypeError:
+            # Django token may have expired time limit 36000
+            django_token = revoke_django_token(client_id, client_secret, django_token)
+            user = get_user_from_token(django_token)
+
+        serializer = SubmitLicenseSerializer(data=request.data)
+        if serializer.is_valid():
+            licenseName = request.data.get('fullname')
+            licenseIdentifier = request.data.get('shortIdentifier')
+            licenseAuthorName = request.data.get('licenseAuthorName')
+            userEmail = request.data.get('userEmail')
+            licenseSourceUrls = [request.data.get('sourceUrl')]
+            licenseOsi = request.data.get('osiApproved')
+            licenseComments = request.data.get('comments')
+            licenseHeader = request.data.get('licenseHeader')
+            licenseText = request.data.get('text')
+            result=request.data.get('result')
+            listVersionAdded = ''
+            licenseNotes = ''
+            result = validate_license_fields(licenseName, licenseIdentifier)
+            if result != '1':
+                return Response({
+                    "result": [result]
+                }, status=status.HTTP_400_BAD_REQUEST)
+
+            xml = generateLicenseXml(licenseOsi, licenseIdentifier, licenseName,
+                        listVersionAdded, licenseSourceUrls, licenseHeader, licenseNotes, licenseText)
+            query = SubmitLicenseModel.objects.create(
+                owner=user,
+                fullname=licenseName,
+                shortIdentifier=licenseIdentifier,
+                licenseAuthorName=licenseAuthorName,
+                userEmail=userEmail,
+                sourceUrl=licenseSourceUrls,
+                osiApproved=licenseOsi,
+                comments=licenseComments,
+                licenseHeader=licenseHeader,
+                text=licenseText,
+                xml=xml,
+                result=result
+            )
+            result = "Failed to submit the license."
+            httpstatus = 201
+            urlType = NORMAL
+            if 'urlType' in request.POST:
+                # This is present only when executing submit license via tests
+                urlType = request.POST["urlType"]
+            statusCode = createIssue(licenseAuthorName, licenseName, licenseIdentifier, licenseComments, licenseSourceUrls, licenseHeader, licenseOsi, token, urlType)
+            returnstatus = str(statusCode)
```

Comment:
Same here, we can reuse the `statusCode` variable instead of making the `returnstatus`

Reply:
Agreed.